tmt/export/yaml.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any, List, Optional
+
+import tmt.base
+import tmt.export
+import tmt.utils
+
+
+@tmt.base.FmfId.provides_export('yaml')
+@tmt.base.Test.provides_export('yaml')
+@tmt.base.Plan.provides_export('yaml')
+@tmt.base.Story.provides_export('yaml')
+class YAMLExporter(tmt.export.ExportPlugin):
+    @classmethod
+    def _export(cls, data: tmt.export._RawExported) -> str:
+        return tmt.utils.dict_to_yaml(data)
+
+    @classmethod
+    def export_fmfid(cls,
```

Comment:
Here the `_collection` suffix is missing, right? But as mentioned in my previous comment, we could have just the collection export methods, so the `_collection` suffix could be removed from all methods below?

Reply:
Fmf id is an exception, and I don't know why, yet. But I dared not to touch it. IIUIC, it's possible to export just an fmf id of an object, rather than the object itself, and it's often used, therefore I merely preserved the behavior until I learn more.

If you think it's more reasonable to go ahead and apply collection here as well, I'd do that, otherwise, I'd add a comment and return to it later.

Reply:
Hm, seems I'm a bit confused: So the `_get_collection_exporter` method is used to get the appropriate exporter for given format. But since the `_get_instance_exporter` method was removed d35bc35 how will this work?

Reply:
True. Can you check (after) 46f8f8ee? It should remove the singular aspect of fmf id - it does seem it's not necessary after all.

Reply:
Yes, thanks, now it makes sense to me.